src/app/components/pages/books/QuantumMechanicsPrimer.tsx

```diff
@@ -1,16 +0,0 @@
-import {Col, Container} from "reactstrap";
-import React from "react";
-import {TitleAndBreadcrumb} from "../../elements/TitleAndBreadcrumb";
-import {PageFragment} from "../../elements/PageFragment";
-
-export const QuantumMechanicsPrimer = () => {
-
-    return <Container className="physics">
-        <Col>
-            <div className="book-intro">
```

Comment:
When removing a whole component, you have also removed the components that were only used by the top component, which is good. But have you also been removing the CSS classes and attributes that are no longer used anywhere else? For instance, that is the case with `book-intro`. It's no longer used, but it's still present in the `.scss` files. I'm not saying that we necessarily need to do this, because it makes sense to keep some of the CSS properties if they're generic enough, or if we think that we will need them in the future, but it's still worth considering. Otherwise you're going to end up with a very messy CSS, that's unnecessarily big and harder to maintain.

Reply:
Good shout, I've not been checking for CSS classes but I will do going forward. Removed this one and a bunch of other "book" related CSS.